pkg/tree/tree.go

```diff
@@ -27,6 +30,29 @@ func (t Tree) String() string {
 	return s
 }
 
+// GetString returns the string value at the given path
+func (t Tree) GetString(path ...string) (string, error) {
+	if len(path) == 1 {
+		// check if exists
+		val, exist := t[path[0]]
+		if exist {
+			// check if string
+			if reflect.TypeOf(val).Kind() == reflect.String {
```

Comment:
instead of using reflect, I think the idiomatic way to do this is with a typesafe cast:

```
if s, ok := val.(string); ok {
  return s, nil
}
return "", fmt.Errorf("expected string but found a different type")
```